modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,136 @@
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users must configure their verification method when they log in to Anypoint Platform after you initially enable multi-factor authentication. If a user already has multi-factor authentication configured for their account, they are not prompted to configure another verification method.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must either configure these calls using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
+--
+
+To enable multi-factor authentication for all user accounts:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Select *Required*.
+. If you want to exempt specific users or service accounts from multi-factor authentication, add them to the *Exempt Accounts* section.
+. Click *Save*. +
+All nonexempt users in your organization are now required to configure a verification method when they log in to Anypoint Platform.
+
+=== Exempt Users from Multi-factor Authentication
+
+Service users or accounts that make programmatic calls to Anypoint Platform should be exempt from multi-factor authentication because they cannot provide verification aside from a username and password.
+
+It is best to use the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. Using the Connected Apps feature provides more security and control than using user accounts.
+
+To exempt individual accounts from multi-factor authentication:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor Auth*.
+. In the *Exempt Accounts* section, add the accounts that you want to exempt. +
+Only users who do not use single sign-on (SSO) appear in this list.
+. Click *Save*. +
+The exempt user accounts appear in the *Exempt Accounts* section.
+
+You can later remove user accounts from the *Exempt Accounts* list by clicking *X* next to their account names.
+
+== Enable Multi-factor Authentication on Your User Account
```

Comment:
```suggestion
== Enable Multi-Factor Authentication on Your User Account
```

Reply:
Are these org users, SSO users, or non-SSO users? 
That's another set of options that I lose track of, along with MFA options, verification options, "you" options....